pkg/network/ebpf/c/tracer/events.h

```diff
@@ -102,13 +102,15 @@ static __always_inline void cleanup_conn(conn_tuple_t *tup, struct sock *sk) {
         return;
     }
 
+    bpf_perf_event_output(ctx, &conn_close_event, cpu, &conn, sizeof(conn));
```

Comment:
Worth moving the comments in line 107-108 to be above that change, and explaining in the code what are you doing here